src/main/java/net/sf/javaanpr/gui/windows/FrameMain.java

```diff
@@ -277,20 +277,25 @@ public RecognizeThread(FrameMain parentFrame) {
             this.parentFrame = parentFrame;
         }
 
+        private void setFailedAndPrintStackTrace(Exception exception) {
+            this.parentFrame.recognitionLabel.setText("failed");
```

Comment:
Optional: Remove all the `this` calls, if you feel like it :) In the spirit of #41